eclipsecs-sevntu-plugin/src/com/github/sevntu/eclipse-metadata.yml

```diff
@@ -0,0 +1,4 @@
+---
+extensionName: "com.github.sevntu"
+ruleGroupName: "SevNTU checks"
+ruleGroupDesc: "Check extensions developed by SevNTU"
```

Comment:
In original config there is some number also  that might be used to define position of extension in tree of modules in eclipse-cs UI

Reply:
Where we can define `priority`
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/83377c1c1d25a95f36837d4b3fa6e721d108a8aa/eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/annotation/checkstyle-metadata.xml#L6

?

Reply:
@romani  I will add a field for it.
But why is `priority` different in some packages whereas rule group is same?
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/83377c1c1d25a95f36837d4b3fa6e721d108a8aa/eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/coding/checkstyle-metadata.xml#L6

Reply:
@Gaurav-Punjabi , it is mistake, it should be the same.

Reply:
> Where we can define `priority`
> https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/83377c1c1d25a95f36837d4b3fa6e721d108a8aa/eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/annotation/checkstyle-metadata.xml#L6
> 
> ?

@romani Done.

Reply:
We should give ability to use multiple groups from single extension in yml, please do

Reply:
I didnt understand why would we need multiple groups, please provide and example/use case

Reply:
> We should give ability to use multiple groups from single extension in yml, please do

@romani Done. The explanation is present in the issue description